crossdock/client/echo/thrift.go

```diff
@@ -42,7 +43,7 @@ func Thrift(t crossdock.T) {
 	fatals.NoError(dispatcher.Start(), "could not start Dispatcher")
 	defer dispatcher.Stop()
 
-	client := echoclient.New(dispatcher.Channel("yarpc-test"))
+	client := echoclient.New(dispatcher.Channel("yarpc-test"), thrift.DisableEnveloping)
```

Comment:
Please add a comment detailing why we need to disable enveloping here. Also, a TODO to remove when we have enveloping support across langs is probably in order.